taken from https://github.com/VDR4Arch/vdr4arch/blob/vdr-stable/plugins/vdr-xmltv2vdr/xmltv2vdr_gcc721_fixes.diff
fixes 

event.cpp: In member function 'void cXMLTVEvent::GetSQL(const char*, int, const char*, char**, char**)':
event.cpp:531:5: error: 'string' was not declared in this scope
     string si=sql_insert;
     ^~~~~~

diff --git a/dist/epgdata2xmltv/epgdata2xmltv.cpp b/dist/epgdata2xmltv/epgdata2xmltv.cpp
index de6fb26..ec465fb 100644
--- a/dist/epgdata2xmltv/epgdata2xmltv.cpp
+++ b/dist/epgdata2xmltv/epgdata2xmltv.cpp
@@ -561,7 +561,7 @@ int cepgdata2xmltv::Process(int argc, char *argv[])
                 enca_analyser_free(analyser);
             }
 
-            string s = xmlmem;
+            std::string s = xmlmem;
             int reps=pcrecpp::RE("&(?![a-zA-Z]{1,8};)").GlobalReplace("%amp;",&s);
             if (reps) {
                 xmlmem = (char *)realloc(xmlmem, s.size()+1);
diff --git a/event.cpp b/event.cpp
index ae33002..1df43ec 100644
--- a/event.cpp
+++ b/event.cpp
@@ -528,7 +528,7 @@ void cXMLTVEvent::GetSQL(const char *Source, int SrcIdx, const char *ChannelID,
         return;
     }
 
-    string si=sql_insert;
+    std::string si=sql_insert;
     int ireps;
     ireps=pcrecpp::RE("'").GlobalReplace("''",&si);
     ireps+=pcrecpp::RE("\\^").GlobalReplace("'",&si);
@@ -540,7 +540,7 @@ void cXMLTVEvent::GetSQL(const char *Source, int SrcIdx, const char *ChannelID,
     }
     *Insert=sql_insert;
 
-    string su=sql_update;
+    std::string su=sql_update;
     int ureps;
     ureps=pcrecpp::RE("'").GlobalReplace("''",&su);
     ureps+=pcrecpp::RE("\\^").GlobalReplace("'",&su);
diff --git a/import.cpp b/import.cpp
index 0d6f7bf..e417c59 100644
--- a/import.cpp
+++ b/import.cpp
@@ -1401,7 +1401,7 @@ bool cImport::UpdateXMLTVEvent(cEPGSource *Source, sqlite3 *Db, cXMLTVEvent *xEv
             return false;
         }
 
-        string ed=shortdesc;
+        std::string ed=shortdesc;
 
         int reps;
         reps=pcrecpp::RE("'").GlobalReplace("''",&ed);
@@ -1511,7 +1511,7 @@ bool cImport::UpdateXMLTVEvent(cEPGSource *Source, sqlite3 *Db, const cEvent *Ev
             return false;
         }
 
-        string ed=eitdescription;
+        std::string ed=eitdescription;
 
         int reps;
         reps=pcrecpp::RE("'").GlobalReplace("''",&ed);
@@ -1649,7 +1649,7 @@ cXMLTVEvent *cImport::SearchXMLTVEvent(sqlite3 **Db,const char *ChannelID, const
             return NULL;
         }
 
-        string st=sqltitle;
+        std::string st=sqltitle;
 
         int reps;
         reps=pcrecpp::RE("'").GlobalReplace("''",&st);
